content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/installation.mdx

```diff
@@ -41,6 +41,25 @@ Then install the Operator:
 $ helm install --version 0.10.0 --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator
 ```
 
+## CSI driver option
+
+You can also use the Vault Secrets Operator Helm chart to deploy a DaemonSet of the
+[Vault Secrets Operator CSI driver](/vault/docs/platform/k8s/vso/csi) with
+the `"csi.enabled"` flag:
+
+```shell-session
+$ helm install                          \
```

Comment:
We already have dedicated installation docs for both new installations and upgrades. Would it make sense to link to those from here?

Reply:
I think this is already the installation docs page that you're talking about. This is just a new small section in that doc, explaining the `"csi.enabled" flag, but the rest is all the same as before. (Unless you're talking about a different doc I don't know about)